Dockerfile

```diff
@@ -129,8 +129,8 @@ COPY build.d common/build.d
 COPY conf.d common/conf.d
 COPY entrypoint.d common/entrypoint.d
 RUN mkdir -p auto/addons custom/src/private \
-    && ln /usr/local/bin/direxec.sh common/entrypoint.sh \
-    && ln /usr/local/bin/direxec.sh common/build.sh \
+    && ln /usr/local/bin/direxec.py common/entrypoint \
+    && ln /usr/local/bin/direxec.py common/build \
```

Comment:
OK, then remove the extension here.

I agree executables under your `$PATH` should have no extension because you should not care about its language. About those under `build.d`, I think those are just scripts whose suffix should not matter much because they should not be executed direcly by the user, but OK, leave them as you wish 😉